app/assets/src/components/ui/controls/GeoSearchInputBox.jsx

```diff
@@ -28,23 +28,31 @@ class GeoSearchInputBox extends React.Component {
 
   // Fetch geosearch results and format into categories for LiveSearchBox
   handleSearchTriggered = async query => {
-    const serverSideSuggestions = await getGeoSearchSuggestions(query);
-    // Semantic UI Search expects results as: `{ category: { name: '', results: [{ title: '', description: '' }] }`
+    let serverSideSuggestions = [];
     let categories = {};
-    if (serverSideSuggestions.length > 0) {
-      const locationsCategory = "Location Results";
-      categories[locationsCategory] = {
-        name: locationsCategory,
-        // Format title/description for the text box.
-        results: serverSideSuggestions.map(r => {
-          const nameParts = r.name.split(", ");
-          r.title = nameParts[0];
-          r.description = nameParts.slice(1).join(", ");
-          r.key = `loc-${r.locationiq_id}`;
-          return r;
-        }),
-      };
+    try {
+      serverSideSuggestions = await getGeoSearchSuggestions(query);
+      // Semantic UI Search expects results as: `{ category: { name: '', results: [{ title: '', description: '' }] }`
+      if (serverSideSuggestions.length > 0) {
+        const locationsCategory = "Location Results";
+        categories[locationsCategory] = {
+          name: locationsCategory,
+          // Format title/description for the text box.
+          results: serverSideSuggestions.map(r => {
+            const nameParts = r.name.split(", ");
+            r.title = nameParts[0];
+            r.description = nameParts.slice(1).join(", ");
+            r.key = `loc-${r.locationiq_id}`;
+            return r;
+          }),
+        };
+      }
+    } catch (e) {
+      // In the case of an error (e.g. no API key in dev), just catch and show the plain text option.
+      // eslint-disable-next-line no-console
+      console.log(e);
```

Comment:
Can you log this particular case somehow? 
It would be important if we are alerted for this and verify the entry... Maybe for now just analytics and include the free text value...

Reply:
@tfrcarvalho added 👍